README.md

```diff
@@ -91,6 +91,13 @@ As a command line argument:
 jupyter notebook --ResourceUseDisplay.track_cpu_percent=True
 ```
 
+### Disable Prometheus Metrics
+There is a [known bug](https://github.com/jupyter-server/jupyter-resource-usage/issues/123) with Prometheus metrics which
+causes "lag"/pauses in the UI. To fix this you can disable Prometheus metric reporting using:
```

Comment:
Since this new option doesn't really "fix" the Prometheus issue, I think it might be better to use "workaround" (or an applicable phrase) so that we don't give the wrong impression:
```suggestion
causes "lag"/pauses in the UI. To workaround this you can disable Prometheus metric reporting using:
```